pkg/debug/scripts/collect-info.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# This script is used to collect all info from the machine which
+# helps to debug and resolve EVE issues.
+#
+
+DATE=$(date -Is)
+INFO_DIR="eve-info-$DATE"
+TARBALL_FILE="/persist/$INFO_DIR.tar.gz"
+
+# Create temporary dir
+TMP_DIR=$(mktemp -d)
+DIR="$TMP_DIR/$INFO_DIR"
+mkdir -p $DIR
+
+# Have to chroot, lsusb does not work from this container
+chroot /hostfs lsusb -vvv    > $DIR/lsusb-vvv
+chroot /hostfs lsusb -vvv -t > $DIR/lsusb-vvv-t
+
+dmesg         > $DIR/dmesg
+ps awux       > $DIR/ps-awux
+lspci -vvv    > $DIR/lspci-vvv
+lspci -vvv -t > $DIR/lspci-vvv-t
+lsblk -a      > $DIR/lsblk-a
+logread       > $DIR/logread
+
+cat /proc/vmallocinfo > $DIR/vmallocinfo
+cat /proc/slabinfo    > $DIR/slabinfo
+cat /proc/zoneinfo    > $DIR/zoneinfo
+cat /proc/mounts      > $DIR/mounts
+cat /proc/vmstat      > $DIR/vmstat
```

Comment:
I think it's a good idea to also include:

To get more hardware information:
- /proc/cpuinfo
- output of dmidecode (not installed, can be added with apk, but it's also available on pillar container, so it should be present on hostfs file system)

Hypervisor config for VMs:
- /run/domainmgr/xen/*.cfg

Doesn't make sense to also include all json files from /run?
- I would do something like this: 
```shell
find /run -name "*.json" | tar -cvjf /tmp/json.tar.bz2 -T -
```

Reply:
Roman has already included the whole /run directory, similarly to what we do for the USB stick "diag" method: https://github.com/lf-edge/eve/blob/master/pkg/pillar/scripts/device-steps.sh#L161-L174

Reply:
That's true @milan-zededa, didn't see it, my bad. But still, I think we are only interested on json files, right? socket files will be ignored, touch files are empty and for pid we can grab from ps output... does it make sense copy the whole /run?

Reply:
@rene yeah, why not to copy the whole /run? 'socket ignored' is used to avoid tar warnings.  Will include the cpuinfo and dmicode.